src/gui/modal.js

```diff
@@ -102,6 +103,65 @@ createNameSpace("realityEditor.gui.modal");
         document.body.appendChild(domElements.fade);
         document.body.appendChild(domElements.container);
     }
+
+    function openInputModal({ headerText, descriptionText, inputPlaceholderText, cancelButtonText, submitButtonText, onCancelCallback, onSubmitCallback, useSmallerVersion }) {
+        // Add a blurry background that can be tapped on to cancel the modal
+        let fade = document.createElement('div'); // darkens/blurs the background
+        fade.id = 'modalFadeClassic';
+        fade.classList.add('modalVisibleFadeIn');
+        document.body.appendChild(fade);
+
+        // create the container with the header, description, a text input, and a submit and cancel button
+        let container = document.createElement('div');
+        container.classList.add('inputModalCard');
+        // container.classList.add('viewCard', 'center', 'popUpModal');
+        let text = document.createElement('div');
+        text.classList.add('inputModalCardText');
+        text.innerHTML = `<h3>${headerText}</h3>
```

Comment:
It would be better to construct the h3 and p elements using createElement and innerText just in case at some point in the future the headerText or descriptionText is provided from an untrusted source